app/guid-file/template.hbs

```diff
@@ -17,6 +17,9 @@
         <div local-class='FlexContainerRow'>
             <h2 data-test-filename>
                 {{this.model.name}}
+                {{#if this.model.providerSpecificData.titleSuffix}}
```

Comment:
Hey Futa, have you considered abstracting this logic to the component typescript file as:  `if (titleSuffix) { // code here}`? It may be preferential in Ember to use the template logic, but may be worth considering.